MapboxNavigation/NavigationViewController.swift

```diff
@@ -213,18 +213,28 @@ open class NavigationViewController: UIViewController {
      
      In cases where you need to update the route after navigation has started you can set a new `route` here and `NavigationViewController` will update its UI accordingly.
      */
-    @objc public var route: Route! {
-        didSet {
-            if routeController == nil {
-                routeController = RouteController(along: route, directions: directions, locationManager: NavigationLocationManager())
-                routeController.delegate = self
-            } else {
-                routeController.routeProgress = RouteProgress(route: route)
-            }
+    @objc public var route: Route {
+        get {
+            return navigationService.route
+        }
+        set {
+            navigationService.route = newValue
             NavigationSettings.shared.distanceUnit = route.routeOptions.locale.usesMetric ? .kilometer : .mile
-            mapViewController?.notifyDidReroute(route: route)
+                        mapViewController?.notifyDidReroute(route: route)
         }
     }
+//    @objc public var route: Route! {
```

Comment:
What's happening here? I'd rather not bring commented-out code into master

Reply:
Apologies, that shouldn't of made it into the PR. 😅